plugins/chat/lib/chat/types/array.rb

```diff
@@ -11,10 +11,20 @@ def cast_value(value)
         case value
         when String
           value.split(",")
+        when ::Array
+          value.map { |item| convert_to_integer(item) }
```

Comment:
I came across this a bit by chance, and I just wanted to share this could be rewritten as:
```ruby
Integer(item, exception: false) || item
```
That’s all :grin:

Reply:
nice thanks, didnt know about exception: false